plugin.json

```diff
@@ -4,6 +4,7 @@
   "description": "Integrates real-time voice communication in Mattermost",
   "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
   "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+  "icon_path": "server/assets/plugin_icon.svg",
```

Comment:
@cpoile Do you know whether this is enough to get the respective Markeplace metadata updated automatically once we publish the release or if we should update that side manually? Thanks.

Reply:
Yes that should be enough -- the marketplace `add` cmd should extract it and encode it for the marketplace.